providers/network/provider/provider.go

```diff
@@ -179,15 +188,16 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 }
 
 func (s *Service) detect(asset *inventory.Asset, conn *connection.HostConnection) error {
-	asset.Name = conn.Conf.Host
+	asset.Name = conn.Conf.Options["host"]
```

Comment:
the host from the options should be set in the conf already at this point

Reply:
Not the case - Where should this mapping happen?